docs/src/main/sphinx/installation/plugins.md

```diff
@@ -0,0 +1,284 @@
+# Plugins 
+
+Trino uses a plugin architecture to extend its capabilities and integrate with
+various data sources and other systems. Trino includes many plugins as part of
+the binary packages - specifically the `tar.gz` archive and the Docker
+container.
+
+The main plugins implement some the following functionalities:
+
+* [Connectors](/connector)
+* [Access control systems](security-access-control)
+* [Event listeners](admin-event-listeners)
+* Additional [types](/develop/types) and [functions](/develop/functions)
+
+All plugins are optional for your use of Trino since they support a specific
```

Comment:
At a minimum, we can remove the comma after functionality, but why not just say "All plugins are optional."

Reply:
nit: prefer _because_ in place of _since_ as a causal conjunction to avoid potential confusion with time-related usage of _since_.

Reply:
Good point. I will still leave the reasoning in place because it acts as an explanation.